src/components/student-dashboard/dashboard.cjsx

```diff
@@ -0,0 +1,80 @@
+React  = require 'react'
+BS     = require 'react-bootstrap'
+moment = require 'moment'
+_ = require 'underscore'
+DontForgetPanel = require './dont-forget-panel'
+EmptyPanel      = require './empty-panel'
+UpcomingPanel   = require './upcoming-panel'
+AllEventsByWeek = require './all-events-by-week'
+ThisWeekPanel   = require './this-week-panel'
+
+{StudentDashboardStore} = require '../../flux/student-dashboard'
+
+module.exports = React.createClass
+  displayName: 'StudentDashboard'
+
+  propTypes:
+    courseId: React.PropTypes.any.isRequired
+
+  contextTypes:
+    router: React.PropTypes.func
+
+  getInitialState: ->
+    selectedTabIndex: 1
+
+  viewFlightPath: ->
+    @context.router.transitionTo('viewGuide', {courseId: @props.courseId})
+
+  selectTab: (index) ->
+    this.setState(selectedTabIndex:index)
+
+  render: ->
+    courseId = @props.courseId
+    info = StudentDashboardStore.get(courseId)
+    {longTitle, shortTitle} = StudentDashboardStore.getTitles(courseId)
+
+    <div className={shortTitle.toLowerCase() + " bg"}>
```

Comment:
if this is used in the LESS file can you document the possible class names here? Also, you can use string interpolation here: `className={"#{shortTitle.toLowerCase()} bg"}`